packages/services/src/models/notification-scanners/AppealsOpened.js

```diff
@@ -0,0 +1,48 @@
+import NotificationScannerBaseModel from './NotificationScannerBaseModel'
+import Network from '@aragonone/court-backend-server/build/web3/Network'
```

Comment:
I wonder if we should lift the common dependencies of the services and server out?

In particular, I'm wondering if we should have something like a `db` package that contains all the migrations, configuration, and models. It would be really weird if the server and services each had separate versions of the models.

Reply:
Creating issue #196